DigitalLearningSolutions.Data.Migrations/202011101344_CompetencyFrameworkDBChanges.cs

```diff
@@ -0,0 +1,172 @@
+﻿namespace DigitalLearningSolutions.Data.Migrations
+{
+    using FluentMigrator;
+    using FluentMigrator.SqlServer;
+    [Migration(202011101344)]
+    public class CompetencyFrameworkDBChanges : Migration
+    {
+        public override void Up()
```

Comment:
I think these changes have slightly broken the service tests 🙂

Reply:
Yes - wasn't expecting them to be part of the pull request so hadn't checked that they were ready - sorry :(

Reply:
If you don't want the framework changes to be included in this PR you can do an [interactive rebase](https://thoughtbot.com/blog/git-interactive-rebase-squash-amend-rewriting-history) and remove the first two commits from this branch (looks like they contain the frameworks work). Just make sure you have those commits on another branch first (e.g. by creating a copy of this branch) so you don't lose them 🙂

Reply:
I'm not confident doing this without losing the commits. I will make sure they are working with the service tests and leave them in here if that's okay?

Reply:
Sure, that's fine 🙂

Reply:
Service tests all pass now :)